include/robot_interfaces/opencv_driver.hpp

```diff
@@ -0,0 +1,77 @@
+/**
+ * @file
+ * @brief Driver to interface with the camera using OpenCV.
+ * @copyright 2020, New York University, Max Planck Gesellschaft. All rights
+ *            reserved.
+ * @license BSD 3-clause
+ */
+
+#pragma once
+
+#include <ctime>
+#include <iostream>
+
+#include <real_time_tools/timer.hpp>
+#include <opencv2/opencv.hpp>
+
+namespace robot_interfaces
+{
+/**
+ * @brief Driver for interacting with any camera using OpenCV.
+ */
+
+template <typename OpenCVObservation>
+class OpenCVDriver
```

Comment:
I just noticed that the driver should actually not be templated as it needs that specific observation type.